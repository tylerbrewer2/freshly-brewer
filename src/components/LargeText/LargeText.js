import styled from 'styled-components';

const LargeText = styled.div.attrs({
  color: props => props.color || 'black'
})`
  font-family: 'FreightBook', 'serif';
  font-size: 48px;
  color: ${props => props.color}
`

export default LargeText;