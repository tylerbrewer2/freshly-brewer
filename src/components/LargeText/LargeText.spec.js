import React from 'react';
import renderer from 'react-test-renderer';

import LargeText from './LargeText';

it('renders correctly', () => {
  const tree = renderer.create(
    <LargeText />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
