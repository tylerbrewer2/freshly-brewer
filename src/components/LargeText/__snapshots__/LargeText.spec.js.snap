exports[`test renders correctly 1`] = `
<div
  className="sc-bdVaJa emyLmK"
  color="#000000" />
`;
